unit-tests/src/test/scala/java/util/concurrent/atomic/AtomicReferenceSuite.scala.disabled

```diff
@@ -0,0 +1,72 @@
+package java.util
+package concurrent
+package atomic
+
+import java.util.function.UnaryOperator
+
+object AtomicReferenceSuite extends tests.Suite {
+
+  // This test suite is INCOMPLETE (obviously!).
+  // It tests the get() method used by the two methods added
+  // today: getAndUpdate() and updateAndGet()
```

Comment:
`today` needs to go as well.

Reply:
@LeeTibbert I think you missed the comment about `today` in the comment. Also, if you are going to change the comment maybe it is better if you just reference the file like follows:
Warning: This class relies on Scala Native being single threaded ...

Reply:
Changed as requested.  Travis is all green.

I simplified the comment and toned down the editorial snark.  
The fewer lines of comment, the fewer places I can get it wrong.

Some comment is necessary to indicate that the code is
scaffolding so that any one with concurrency training & experience
reading the code does not waste time.  Appropriate precision.

@ekrich  I built on your suggestion and made the comment in the source file 
minimal.  Thank you.